tendrl/notifier/notification/__init__.py

```diff
@@ -51,11 +53,11 @@ def dispatch_notification(self, msg):
         raise NotImplementedError()
 
 
-class NotificationPluginManager(object):
+class NotificationPluginManager(gevent.greenlet.Greenlet):
```

Comment:
Did this work??
I remember having faced some issue around this..
Just verify once